src/__tests__/cucumber_tests/shared_steps/equalizerApo.ts

```diff
@@ -0,0 +1,24 @@
+import { DefineStepFunction } from 'jest-cucumber';
+import { flush, IState } from 'main/flush';
+import { getConfigPath, isEqualizerAPOInstalled } from 'main/registry';
+
+export const givenEqualizerApoIsInstalled = (given: DefineStepFunction) => {
+  given('EqualizerAPO is installed', async () => {
+    if (!(await isEqualizerAPOInstalled())) {
+      throw new Error('EqualizerAPO not installed');
+    }
+    // TODO find a way to install EqualizerAPO
+  });
+};
+
+export const givenCanWriteToAquaConfig = (given: DefineStepFunction) => {
+  given('Aqua can write to Aqua config', async () => {
```

Comment:
this is out of scope for this PR but would we be able to leverage this given to manually set the initial state of the equalizer?